### PR TITLE
Prepare CI for tang operator

### PR DESCRIPTION
**Pull request by Koncpa**

### CODE REVIEW

**Review by sarroutbi — CHANGES_REQUESTED** [review]

File: Setup/setup_local_cluster/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+MINIKUBE_URL="https://storage.googleapis.com/minikube/releases/latest"
+MINIKUBE_FILE="minikube-latest.x86_64.rpm"
+MINIKUBE_URL_FILE="${MINIKUBE_URL}/${MINIKUBE_FILE}"
+OLM_INSTALL_TIMEOUT="5m"
+OPERATOR_SDK_DL_URL="https://github.com/operator-framework/operator-sdk/releases/download/v1.12.0"
```
[the remark this answers is not in the document]
Comment (Koncpa) [reply Setup/setup_local_cluster/runtest.sh:22]: Yeah, thanks leftover..

File: Setup/setup_local_cluster/runtest.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+rlJournalStart
+
+    rlPhaseStartTest "Main test"
+          #install operator sdk
+          ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n "$(uname -m)" ;; esac)
+          OS=$(uname | awk '{print tolower($0)}')
+          OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.12.0
```
[the remark this answers is not in the document]
Comment (Koncpa) [reply Setup/setup_local_cluster/runtest.sh]: Sure, I adjust setup to download latest `operator-sdk`..

**Review by sarroutbi — APPROVED** [review]
Changes LGTM

**Review by sarroutbi — COMMENTED** [review]
Please, if possible, include a `Resolves: #5` in commit, so that it closes the issue regarding Github CI action

File: Sanity/main.fmf [hunk 3]
```diff
@@ -7,6 +7,8 @@ test: ./runtest.sh
 recommend:
   - tang
   - clevis
+require:
+  - jq
```
Comment (sarroutbi) [inline Sanity/main.fmf:11]: May I ask to also require helm and podman, so that non-functional tests are also executed?

Comment (Koncpa) [reply Sanity/main.fmf:11]: Sure, I'll add there these requires.

**Pull request merged**